Source/WebKit/Platform/IPC/StreamServerConnectionBuffer.h

```diff
@@ -59,7 +59,7 @@ class StreamServerConnectionBuffer : public StreamConnectionBuffer {
 
 inline std::optional<StreamServerConnectionBuffer> StreamServerConnectionBuffer::map(Handle&& handle)
 {
-    auto sharedMemory = WebKit::SharedMemory::map(handle.memory, WebKit::SharedMemory::Protection::ReadWrite);
+    auto sharedMemory = WebKit::SharedMemory::map(WTFMove(handle.memory), WebKit::SharedMemory::Protection::ReadWrite);
```

Comment:
So here StreamServerConnectionBuffer::Handle is a function where the author already thought that handles have to have move semantics as the consumption operation, and thus already declared Handle as && so you don't need to do the same thing as with ShareableBitmap/ShareableResource.